lib/app/views/languages/languages.erb

```diff
@@ -5,6 +5,12 @@
 	<% if active %>
 		<article class="article-block">
 			<div class="article-contents">
+				<% if docs[:docs][:about].empty? == false %>
```

Comment:
I had an about method but for some reason I kept getting a undefined method error. So I went with the hash. I'll try again.